src/harness/certs/README.md

```diff
@@ -24,6 +24,13 @@ https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/
                    sas_expired                                               domain_proxy_expired
                                                                              domain_proxy_inapplicable
 
+                  -------------root_ca (same as above, separate graph for the clarity)--------------------
+                 /                    \         		     \
```

Comment:
Please remove the tab characters from this diagram.

Reply:
Typo has   been fixed